src/sql/src/func.rs

```diff
@@ -2302,6 +2302,9 @@ pub static PG_CATALOG_BUILTINS: Lazy<BTreeMap<&'static str, Func>> = Lazy::new(|
         "pg_backend_pid" => Scalar {
             params!() => UnmaterializableFunc::PgBackendPid => Int32, 2026;
         },
+        "pg_cancel_backend" => Scalar {
+            params!(Int32) => Operation::unary(|_ecx, _e| bail_unsupported!("pg_cancel_backend in this position"))  => Bool, 2171;
```

Comment:
Having this error message is great. Otherwise we'd gaslight users about whether this function is real or not, depending on how they type in their SQL queries.